lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -640,6 +640,9 @@ def _delete_old_flows(self, records):
             if current_ver == record.rule_version:
```

Comment:
I've initially added this for the drop flow rule case. The current_ver should be 0 and record.rule_version is also 0.

Reply:
@koolzz I added logs just after this condition

+                self.logger.error('delete rule %s %s %s', record.rule_id, record.rule_version, current_ver)

ERROR:EnforcementStatsController:delete rule internal_default_drop_flow_rule 0 -1
ERROR:EnforcementStatsController:delete rule internal_default_drop_flow_rule 0 -1

The added condition does not catch up the default drop flow.

Reply:
ahh right I see what happened, basically initially it was 0, but when we were updating some rule cleanup the default return on rule not found changed to -1 on this pr https://github.com/magma/magma/commit/8dbf40f29cfe71084d1778f8d59d6429cf9c0f21.

Which accidentally broke this logic.
@pshelar  is -1 required, or should we just go back to returning 0 in `get_version`?

Reply:
ok, gets revert to return zero.
Can you also add comment explaining why it needs to be zero.